jbehave-support-core/src/main/java/org/jbehavesupport/core/ssh/SshSteps.java

```diff
@@ -133,6 +182,61 @@ public void dataNotInLog(String systemQualifier, String stringTable) {
         checkDataPresence(systemQualifier, scenarioStart, stringTable, notContainsVerifier);
     }
 
+    @Given("set ssh reporter mode to [$mode]")
+    public void setSshReporterMode(ExpressionEvaluatingParameter<String> mode) {
+        if (serverLogXmlReporterExtension != null) {
+            serverLogXmlReporterExtension.setSshReportType(SshReportType.valueOf(mode.getValue()));
+        } else {
+            log.warn("ServerLogXmlReportExtension is not registered, no logs will be present in report.");
+        }
+    }
+
+    /**
+     * This step is just for testing of serverLogXmlReporterExtension
```

Comment:
If just for testing then I would put it in some test specific file like groovy test config or someplace like that.

Reply:
I was removed completly with CUSTOM mode